cv2/Error.pyi

```diff
@@ -0,0 +1,110 @@
+BadAlign = -21
```

Comment:
Simply that they exist.

Reply:
> Simply that they exist.

I haven't written stubs for a C/C++ library before, so maybe I'm missing something.

I don't see the ALL_CAPS versions in the [OpenCV docs](https://docs.opencv.org/3.4/d1/d0d/namespacecv_1_1Error.html) or the C++ implementation. Are you suggesting that they should exist because of the Python CONSTANT [naming convention](https://peps.python.org/pep-0008/#constants)? That seems weird to me since it doesn't match the OpenCV documentation, which I'm assuming `opencv-python` users rely heavily on, and the rest of the API names already don't align with Python naming conventions (ex. `cubeRoot` instead of `cube_root`).

Reply:
I'm saying that they exist at runtime, the following is all valid code:
```python
>>> from cv2 import Error
>>> Error.BadAlign
-21
>>> Error.BAD_ALIGN
-21
```
Which convention users decide to take is up to them.